apis/python/src/tiledbsc/scgroup.py

```diff
@@ -249,6 +249,9 @@ def write_X(self, anndata: ad.AnnData):
             sparse=True,
             allows_duplicates=True,
             offsets_filters=[tiledb.DoubleDeltaFilter(), tiledb.BitWidthReductionFilter(), tiledb.ZstdFilter()],
+            capacity=100000,
+            cell_order='row-major',
+            tile_order='col-major',
```

Comment:
Whats the rational for col-major ordering of the tiles?

Reply:
Info from @bkmartinjr on Slack ...

Reply:
Our own access patterns require a balance between two access patterns and we empirically found this is the best config.  

In our use cases, the dominant slices are:
    X[lots_of_scattered_row, :]
and
    X[:, small_number_of_columns]

Happy to share details if helpful.

I suspect that this is yet another hint that the API needs to allow caller-specified schema hints

Reply:
> I suspect that this is yet another hint that the API needs to allow caller-specified schema hints

> it's not strictly necessary to use that here as a default --_ soon (next day or two, 🤞) we'll support exposing tiledb schema parameters to the tiledbsc-r user via args in the soma-constructor -- this can wait until then.

@bkmartinjr 100% we are on the same page :)

Reply:
Thanks for the details @bkmartinjr , I agree making it user configurable will be best long term. For now, happy to accept this!